### PR TITLE
Add repo validation test class

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

File: marketplace/README.md [hunk 1]
```diff
@@ -48,3 +48,16 @@ A repository can be validated using the `MarketplaceClient`. The client pulls a
         e.printStackTrace();
     }
 ```
+
+Note that in this example we have used the default `SignatureType.BASE64_ENCODED` which specifies that the signature files are
+base64 encoded. If you require non-base64 encoded use `SignatureType.SIG`.
```
Comment (karianna) [inline marketplace/README.md:53]: Weird line space wrapping here and below?

File: marketplace/adoptium-marketplace-client/src/test/java/net/adoptium/marketplace/client/RepoTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public void validateRepo() throws IOException {
+        String publicKey = Files.readString(Path.of("../exampleRepositories/keys/public.pem"));
+
+        String repoUrl = "http://localhost:8000/";
+
+        try {
+            MarketplaceClient client = MarketplaceClient.build(repoUrl, SignatureType.BASE64_ENCODED, publicKey);
+            ReleaseList releaseList = client.readRepositoryData();
+
+            System.out.println("Found: " + releaseList.getReleases().size() + " releases");
```
Comment (karianna) [inline marketplace/adoptium-marketplace-client/src/test/java/net/adoptium/marketplace/client/RepoTest.java:26]: system out and system err?  Not a logging framework?

**Review by karianna — CHANGES_REQUESTED** [review]

File: marketplace/adoptium-marketplace-client/src/test/java/net/adoptium/marketplace/client/RepoTest.java (hunk 2 above)
Comment (johnoliver) [reply marketplace/adoptium-marketplace-client/src/test/java/net/adoptium/marketplace/client/RepoTest.java:26]: Going for a compact example with as few dependencies as possible as this is to some extent code I expect vendors to copy and paste to validate their repos

File: marketplace/README.md (hunk 1 above)
Comment (johnoliver) [reply marketplace/README.md:53]: think its fine

**Pull request merged**